### PR TITLE
Add statusbar and remove scenegraph title

### PR DESCRIPTION
**Pull request by fernandojsg**
Related to: https://github.com/aframevr/aframe-editor/issues/147

### CODE REVIEW

File: src/components/Statusbar.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var React = require('react');
```
Comment (ngokevin) [inline src/components/Statusbar.js]: `import React from 'react';`

File: src/components/Statusbar.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import NumberWidget from './widgets/NumberWidget';
+
+export default class Statusbar extends React.Component {
```
Comment (ngokevin) [inline src/components/Statusbar.js]: Use a stateless component

File: src/components/Statusbar.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+var React = require('react');
+import NumberWidget from './widgets/NumberWidget';
```
Comment (ngokevin) [inline src/components/Statusbar.js]: Unused

File: src/components/Main.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ import AttributesSidebar from './attributes/AttributesSidebar';
 import {MenuWidget} from './menu/Menu';
 import ModalTextures from './modals/ModalTextures';
 import Scenegraph from './Scenegraph';
+import Statusbar from './Statusbar';
```
Comment (ngokevin) [inline src/components/Main.js]: Should camelCase

File: src/css/dark.css [hunk 5]
```diff
@@ -227,26 +227,23 @@ input.number:focus {
     background-color: #111;
   }
 
-#toolbar {
+#statusbar {
   position: absolute;
-  left: 0;
-  right: 330px;
+  left: 0px;
```
Comment (ngokevin) [inline src/css/dark.css]: Could just do `0`

**Comment by ngokevin** [comment]
I'd probably reverse the order of the actions: `<DELETE> <ADD>`. You usually want the positive action on the right hand side.

**Comment by ngokevin** [comment]
Looks good after that

**Comment by dmarcos** [comment]
Also you want the most common operations first. And you want to add friction to destructive and non reversible actions. Being on the right makes more unlikely to be clicked by accident. Anyways, I think the delete icon should be hidden away in a contextual menu for the entities.

**Comment by fernandojsg** [comment]
Yes the delete button should probably be replaced by some of the following alternatives:
- Add an option in `Menu > Edit > Remove entity`
- When clicked on the scenegraph, show the delete icon aligned to the right on the same line of the entity.
- Contextual menu over the entity both on the viewport and on the scenegraph.
- `Backspace` shortcut

File: src/components/Statusbar.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+export default class Statusbar extends React.Component {
+  render() {
+    return <div id='statusbar'>
```
Comment (cvan) [inline src/components/Statusbar.js]: also add `className='statusbar'` so we don't have to do a bunch of `!important`s in the stylesheet

**Comment by fernandojsg** [comment]
Transforms gizmos:
https://www.dropbox.com/s/zck3dfwqyrunwto/transforms_gizmos.mov?dl=0

And filters:
https://www.dropbox.com/s/x1715fjtkz239q3/filter_entities.mov?dl=0

Current style:
screenshot 2016-07-08 18 18 33

**Comment by fernandojsg** [comment]
Please take a look at it, and if it's ok for you we could merge and continue the next features in another PR, not to mix many things on the same PR. In fact I could have split search & transforms in two PR, if you think it could help the review I'll do that! :)

**Comment by ngokevin** [comment]
Yeah, can we split?

**Comment by fernandojsg** [comment]
@ngokevin sure! First one with the filters: https://github.com/aframevr/aframe-editor/pull/172

**Comment by fernandojsg** [comment]
@ngokevin and the second one https://github.com/aframevr/aframe-editor/pull/173
I close this PR

**Pull request closed without merging**